### PR TITLE
Feat/cellar stats improvements

### PR DESCRIPTION
**Pull request by codingki**
Cellar stats adjustments

## Description

- remove asset symbol on charts
- bring back 1D change
- add TVM on landing pages

## Changes

- [x] [feat: remove symbol asset on charts](https://github.com/strangelove-ventures/sommelier/commit/a808152d114039199f9bcfc370e37f2e3048308c)
- [x] [feat: change weekly to daily](https://github.com/strangelove-ventures/sommelier/commit/3066d034cf62bd1d842b56a7f4ef008ba6a48ed4)
- [x] [chore: add more ui config](https://github.com/strangelove-ventures/sommelier/commit/d7681edb6ade5eee48f5853ff1af5b8c550b27f2)
- [x] [refactor: conditional ui using cellarType](https://github.com/strangelove-ventures/sommelier/commit/004e4109033989ab80dac3516ccf8543203a4b9b)
- [x] [refactor: remove unnecessary props](https://github.com/strangelove-ventures/sommelier/commit/c204f9a7fda1b8a0a1ad1995b6d0ddea35000847)
- [x] [refactor: cellar cards and adjust information](https://github.com/strangelove-ventures/sommelier/commit/aca84112b749a1425a853a6a24b3d3bc648b53d3)
- [x] [feat: enable tvm and daily change on strategy landing pages](https://github.com/strangelove-ventures/sommelier/commit/811d32af0474aca76b594096c7263741fe980d71)
- [x] [feat: enable daily change](https://github.com/strangelove-ventures/sommelier/commit/bcc0aed2f61ffc9be155cc7f105a78a1860e14aa)
- [x] [feat: move tvm position](https://github.com/strangelove-ventures/sommelier/commit/aa7981c06360b083aa8aa1d0f210ca65c0baaef2)

Screenshot: 
CleanShot 2022-11-08 at 05 47 03@2x
CleanShot 2022-11-08 at 05 47 34@2x

### CODE REVIEW

**Review by xanderjl — CHANGES_REQUESTED** [review]
Really solid improvements! Just missed a few commented out lines, but once those are cleared out, this is good to go! Excellent work @codingki 💪

File: src/components/CellarStatsAutomated.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { ReactNode, VFC } from "react"
 import {
+  Box,
   // Box,
```
[the remark this answers is not in the document]
Comment (codingki) [reply src/components/CellarStatsAutomated.tsx:4]: nice catch

File: src/components/HeroStrategy/hero-right.tsx [hunk 2]
```diff
@@ -7,22 +7,21 @@ import {
   Stack,
   StackDivider,
   Text,
-  Tooltip,
   useDisclosure,
   VStack,
 } from "@chakra-ui/react"
 // import { PercentageText } from "components/PercentageText"
```
[the remark this answers is not in the document]
Comment (codingki) [reply src/components/HeroStrategy/hero-right.tsx:13]: nice catch!

**Review by joshuanatanielnm — APPROVED** [review]
LGTM, But why not create new hooks instead of update the 1W changes. I think we gonna need to show that data latter

**Comment by codingki** [comment]
Good suggestion but I'm thinking that we can set the range dynamically so we are not creating hooks every range for future improvements

**Pull request merged**